pool/app/message_template/entity.ml

```diff
@@ -7,6 +7,27 @@ end
 module Label = struct
   let go m fmt _ = Format.pp_print_string fmt m
 
+  (** ConfirmationWithoutSelfRegistration
```

Comment:
What about this comment?

Reply:
These are the translations i removed from I18n that looked like they were meant to be emails. Will have to figure out when they should be sent and what the content should be.

Reply:
1. ConfirmationWithoutSelfRegistration
    - This is currently handled when an assignment is created (in all cases the same)
    - could be removed for now
1. ImportInvitation
    - Don't know about this
1. InvitationWithoutSelfRegistrationSubject
    - title for the first one
1. ExperimentFinish
    - Notification to the Experimenter/Recruiter that the experiment is finished and they should finalize all data.
1. SessionFinish
    - Notification to the Assistant/Experimenter to complete the sessions

Reply:
I will ask andi in the afternoon.

There were
| ConfirmationWithoutSelfRegistrationSubject & ConfirmationWithoutSelfRegistrationText
| InvitationWithoutSelfRegistrationSubject & InvitationWithoutSelfRegistrationText

So probably this would be the templates sent when going through the waiting list process.

Reply:
created an issue